setup.py

```diff
@@ -7,6 +7,7 @@
 
 requires = ['jmespath==0.7.1',
             'python-dateutil>=2.1,<3.0.0',
+            'rsa>=3.1.2,<=3.1.4',
```

Comment:
This is weird.  Why are we locking to 3 patch version?  Either lock to a minor version or a major version.